code/src/components/QuestionThree.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import myImage from '../assets/boss.png'
+import myImage2 from '../assets/boss2.webp'
+import myImage3 from '../assets/boss3.webp'
+// eslint-disable-next-line no-unused-vars
+const bossMemes = ['1.Dying', '2.Flexible', '3.Friday'];
+
+// eslint-disable-next-line max-len
+const QuestionThree = ({ questionThree, bossMeme, setBossMeme, onStepChange, onQuestionThreeInputChange }) => {
```

Comment:
I dont think the `questionThree` props is necessary here? As you have setQuestionThree={setQuestionThree} and questionThree={questionThree} in the Survey.js.